test/docgen/view/__snapshots__/struct.test.ts.snap

```diff
@@ -195,6 +195,201 @@ public CfnTag[] Tags { get; set; }
+      },
+      "usage": "RepositoryName *string",
+    },
+    Object {
+      "default": undefined,
+      "displayName": "RepositoryPolicyText",
+      "docs": Object {
+        "links": Array [
+          "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-publicrepository.html#cfn-ecr-publicrepository-repositorypolicytext",
+        ],
+        "summary": "\`AWS::ECR::PublicRepository.RepositoryPolicyText\`.",
+      },
+      "fqn": "github.com/aws/aws-cdk-go/awscdkawsecr.CfnPublicRepositoryProps.property.RepositoryPolicyText",
+      "id": "@aws-cdk/aws-ecr.CfnPublicRepositoryProps.property.repositoryPolicyText",
+      "optional": true,
+      "type": Object {
+        "formattingPattern": "interface{}",
+      },
+      "usage": "RepositoryPolicyText interface{}",
+    },
+    Object {
+      "default": undefined,
+      "displayName": "Tags",
+      "docs": Object {
+        "links": Array [
+          "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-publicrepository.html#cfn-ecr-publicrepository-tags",
+        ],
+        "summary": "\`AWS::ECR::PublicRepository.Tags\`.",
+      },
+      "fqn": "github.com/aws/aws-cdk-go/awscdkawsecr.CfnPublicRepositoryProps.property.Tags",
+      "id": "@aws-cdk/aws-ecr.CfnPublicRepositoryProps.property.tags",
+      "optional": true,
+      "type": Object {
+        "formattingPattern": "*[]%",
+        "types": Array [
+          Object {
+            "formattingPattern": "%",
+            "types": Array [
+              Object {
+                "displayName": "CfnTag",
+                "fqn": "undefined/awscdkcore.CfnTag",
+                "id": "@aws-cdk/core.CfnTag",
+                "packageName": "@aws-cdk/core",
+                "packageVersion": "99.99.99",
+              },
+            ],
+          },
+        ],
+      },
+      "usage": "Tags *[]CfnTag",
+    },
+  ],
+  "usage": "import \\"github.com/aws/aws-cdk-go/awscdkawsecr\\"
+
+&awscdkawsecr.CfnPublicRepositoryProps {
+	RepositoryCatalogData: interface{},
+	RepositoryName: *string,
+	RepositoryPolicyText: interface{},
+	Tags: *[]undefined/awscdkcore.CfnTag,
+}",
+}
+`;
+
+exports[`go snapshot 2`] = `
+" CfnPublicRepositoryProps <a name=\\"CfnPublicRepositoryProps\\" id=\\"@aws-cdk/aws-ecr.CfnPublicRepositoryProps\\"></a>
+
+Properties for defining a \`AWS::ECR::PublicRepository\`.
+
+> [http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-publicrepository.html](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-publicrepository.html)
+
+# Initializer <a name=\\"Initializer\\" id=\\"@aws-cdk/aws-ecr.CfnPublicRepositoryProps.Initializer\\"></a>
+
+\`\`\`go
+import \\"github.com/aws/aws-cdk-go/awscdkawsecr\\"
+
+&awscdkawsecr.CfnPublicRepositoryProps {
+	RepositoryCatalogData: interface{},
+	RepositoryName: *string,
+	RepositoryPolicyText: interface{},
+	Tags: *[]undefined/awscdkcore.CfnTag,
+}
```

Comment:
I see how this a bit different / unexpected compared to what go devs (gophers?) are probably used to. I definitely think it could be a nice future enhancement to align with pkg.go.dev